templates/user/profile.tmpl

```diff
@@ -90,14 +90,19 @@
 			</div>
 			<div class="ui eleven wide column">
 				<div class="ui secondary stackable pointing tight menu">
-					<a class='{{if and (ne .TabName "activity") (ne .TabName "following") (ne .TabName "followers") (ne .TabName "stars") (ne .TabName "watching") (ne .TabName "projects")}}active{{end}} item' href="{{.Owner.HomeLink}}">
+					<a class='{{if and (ne .TabName "activity") (ne .TabName "following") (ne .TabName "followers") (ne .TabName "stars") (ne .TabName "watching") (ne .TabName "projects") (ne .TabName "code")}}active{{end}} item' href="{{.Owner.HomeLink}}">
 						{{svg "octicon-repo"}} {{.i18n.Tr "user.repositories"}}
 					</a>
 					{{if .IsPackageEnabled}}
 					<a class='{{if eq .TabName "packages"}}active{{end}} item' href="{{.Owner.HomeLink}}/-/packages">
 						{{svg "octicon-package"}} {{.i18n.Tr "packages.title"}}
 					</a>
 					{{end}}
+					{{if .IsRepoIndexerEnabled}}
+					<a class='{{if eq .TabName "code"}}active{{end}} item' href="{{.Owner.HomeLink}}/-/code">
```

Comment:
To not conflict with "code" quotes (it same as in packages 5 lines above

Reply:
But Go has no problems with double quotes. See https://github.com/go-gitea/gitea/pull/19977/files#diff-74d7b797dadbdf4ffadfd7850771db41d2fcad899612d1c47ab2fa972f214269R3

Reply:
imho it just for code editor highlighting to not get lost. I just kept same style as in other places in this file. It would be worse to use different code style in same file. And changing in all file would be totally out of scope for this PR